alternative_backend/orders/models.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models
+
+
+class Client(models.Model):
+    name = models.CharField(max_length=100)
+    country = models.CharField(max_length=50)
+    city = models.CharField(max_length=50)
+    post_code = models.CharField(max_length=20)
+    adress = models.CharField(max_length=100)
+    is_company = models.BooleanField()
+
+    def __str__(self):
+        return '%s %s' % (self.name, self.country)
+
+    class Meta:
```

Comment:
Przesuń to na samą górę. `class Meta` to ważna informacja i powinna być na początku.